statsmodels/sandbox/distributions/tests/test_norm_expan.py

```diff
@@ -20,7 +21,8 @@
 
 class CheckDistribution(object):
 
-    def test_smoke(self):
+    @pytest.mark.smoke
+    def test_smoke(self):  # TODO: better name
```

Comment:
AFAICS this is not a smoke test
it is a round trip test with the corresponding asserts
(test name is most likely from when it only was a smoke test)

Reply:
It looks like the first two lines are a smoke test for dist1 and the rest are effectively a separate test?

Reply:
updated